server/api/25words.js

```diff
@@ -34,23 +34,23 @@ router.post("/", async (req, res, next) => {
       // find 1 int from 0 1 2 3
       const randomInt = getRandomIntArray(1, 4);
```

Comment:
Here it might be cleaner to either use a new helper function that just gets a singular, random int inbetween 0 and 4 inclusively,

Taken from: https://www.w3schools.com/JS/js_random.asp
```js
function getRndInteger(min, max) {
  return Math.floor(Math.random() * (max - min + 1) ) + min;
}
```

or, if you want to use the helper func we've written already, we can change this line to:
```js
const randomInt = getRandomIntArray(1, 4)[0];
```

Because having `randomInt` be an array of a singular random int that we then have to always index at 0 is confusing I think